### PR TITLE
Require pylint_plugin_utils and use setuptools

### PR DESCRIPTION
**Pull request by jakirkham**
Fixes https://github.com/landscapeio/pylint-common/issues/2

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -34,6 +33,7 @@
 else:
     _install_requires = [
         'pylint>=1.0',
+        'pylint-plugin-utils>=0.2.4',
```
Comment (jakirkham) [inline setup.py]: Not sure if this is a reasonable version requirement, but it appears to be the most recent and has been around for nearly a year.

**Comment by jakirkham** [comment]
If you are thinking about releasing, @carlio, I think we might want to fix this first.

File: setup.py [hunk 2]
```diff
@@ -30,10 +29,12 @@
         'pylint>=1.0,<1.4',
         'astroid>=1.0,<1.3.0',
         'logilab-common>=0.60.0,<0.63',
+        'pylint-plugin-utils>=0.2.4',
```
Comment (jakirkham) [inline setup.py]: Also same story with this version constraint.

File: setup.py (hunk 1 above)
Comment (carlio) [inline setup.py]: Just made a new minor release of that too to include the LICENSE file there as well but it otherwise hasn't changed in years!

**Comment by carlio** [comment]
Merged and released, thanks! 0.2.4 living on pypi now.

**Pull request merged**